packages/gatsby-transformer-remark/src/__tests__/extend-node.js

```diff
@@ -245,7 +245,7 @@ title: "my little pony"
 date: "2017-09-18T23:19:51.246Z"
 ---
 
-Where oh [*where*](nick.com) **_is_** that pony?`,
+Where oh [*where*](nick.com) **_is_** ![pony.png](that pony)?`,
```

Comment:
Did you do the switch of filename and alt tag on purpose (pony)?

Reply:
hah, I did not. Good catch!

i'll fix it right away

Reply:
fixed it :)